ui/components/ui/show-hide-toggle/index.scss

```diff
@@ -0,0 +1,34 @@
+.show-hide-toggle {
+  position: relative;
+  display: inline-flex;
+
+  &__input {
+    appearance: none;
+
+    + .show-hide-toggle__label {
+      cursor: pointer;
+      user-select: none;
+    }
+
+    /* Focused when tabbing with keyboard */
+    &:focus,
+    &:focus-visible {
+      outline: none;
+
+      + .show-hide-toggle__label {
+        outline: Highlight auto 1px;
```

Comment:
Where does `Highlight` come from?  Never seen this before, would love to know what the reference is!

Reply:
Oh, my bad! Should have left a comment for this. This is a "System color": https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#system_colors

Reply:
And here's the W3C spec for it: https://www.w3.org/TR/css-color-4/#css-system-colors

I saw a few references to this color having been deprecated, but it seems that W3C went ahead and un-deprecated it. So it should be safe to use. It works on Chrome and Firefox as well.